tezos_interop/tezos_interop.re

```diff
@@ -40,7 +50,11 @@ module Ed25519 = {
     let hash_key = t =>
       BLAKE2B_20.hash(Ed25519.pub_to_cstruct(t) |> Cstruct.to_string);
 
-    // TODO: encoding
+    let encoding = {
+      let name = "Ed25519.Public_key_hash";
+      // TODO: in tezos this is splitted json is not same as bin
```

Comment:
I'm not sure what this comment means, could it be clarified?

Reply:
BTW not a big deal, but "splitted" isn't an english word, it's just "split"

Reply:
"splitted" seems to be a valid word in english but it is copied from tezos

https://gitlab.com/tezos/tezos/-/blob/master/src/lib_crypto/helpers.ml#L134

https://www.merriam-webster.com/dictionary/splitted

Reply:
@anchpop do you have any suggestion on the comment that should go here?